contributors/design-proposals/customresources-subresources.md

```diff
@@ -127,16 +127,16 @@ validation:
 
 Moreover, if the scale subresource is enabled:
 
-- The value at the specified JSON Path `SpecReplicasPath` (e.g. `.spec.replicas`) is validated to ensure that it contains non-negative integer value and is not empty.
+On update, we copy the values from the `Scale` object into the specified paths in the CustomResource, if the path is set (`StatusReplicasPath` and `LabelSelectorPath` are optional).
+If `StatusReplicasPath` or `LabelSelectorPath` is not set, we validate that the value in `Scale` is also not specified and return an error otherwise.
 
-- The value at the optional JSON Path `StatusReplicasPath` (e.g. `.status.replicas`) is validated to be a an integer number if it exists (i.e. this can be empty).
+On get and on update (after copying the value into the CustomResource as described above), we verify that:
 
-- The value at the optional JSON Path `LabelSelectorPath` (e.g. `.spec.labelSelector`) is validated to be a valid label selector if it exists (i.e. this can be empty).
+- The value at the specified JSON Path `SpecReplicasPath` (e.g. `.spec.replicas`) is a non-negative integer value and is not empty.
 
-If one of the path fields in `CustomResourceSubResourceScale`:
-- does not exist within the CR, it maps to the default value (0 for the replicas and empty for the label selector).
-- exists within the CR but it does not validate (no integer or no label selector), we return an error.
+- The value at the optional JSON Path `StatusReplicasPath` (e.g. `.status.replicas`) is an integer value if it exists (i.e. this can be empty).
 
+- The value at the optional JSON Path `LabelSelectorPath` (e.g. `.spec.labelSelector`) is a valid label selector if it exists (i.e. this can be empty).
```

Comment:
I wonder, the same validation should be done when the whole CR is updated and when .status is updated. Maybe we just add this here with another sentence.

Reply:
won't we end up validating spec with status updates then?

Reply:
but we should mention at least about `statusReplicasPath`